modules/apps/foundation/frontend-theme/justin-theme/src/js/main.js

```diff
@@ -0,0 +1,85 @@
+AUI().ready(
+
+	/* This function gets loaded when all the HTML, not including the portlets, is loaded. */
+
+	function() {
+
+		var delta = 10;
+		var lastScrollTop = 0;
+		var navbarHeight = $('#banner').outerHeight();
+		var scrolling;
+
+		$(window).scroll(
+			function(event) {
+				scrolling = true;
+			}
+		);
+
+		setInterval( function() {
+			if (scrolling) {
+				moveBar();
+				scrolling = false;
+			}
+		}, 250);
+
+		function moveBar() {
+			var st = $(this).scrollTop();
+
+		    // Make sure they scroll more than delta
+
+			if (Math.abs(lastScrollTop - st) <= delta) {
+				return;
+			}
+
+		    // If they scrolled down and are past the navbar, add class .nav-up.
+
+		    // This is necessary so you never see what is 'behind' the navbar.
+
+			if (st > lastScrollTop && st > navbarHeight) {
+
+		        // Scroll Down
+
+				$('#banner').removeClass('nav-down').addClass('nav-up');
```

Comment:
Can't we use ```switchClass()```?

Reply:
It appears that Liferay is using jquery 2.1.4, which does not have `switchClass()`

https://code.jquery.com/jquery-2.1.4.js

Reply:
Oh then keep this as is.  I think we might have updated it in a newer version of Liferay.